swap_meet/item.py

```diff
@@ -1,2 +1,31 @@
+from uuid import uuid4
+
+CONDITION_DESCRIPTION = { 0: "heavily used", 
```

Comment:
Nitpick: extra whitespace should be removed
```suggestion
CONDITION_DESCRIPTION = {0: "heavily used", 
```